aiohttp/web_fileresponse.py

```diff
@@ -37,6 +39,14 @@
 
 NOSENDFILE: Final[bool] = bool(os.environ.get("AIOHTTP_NOSENDFILE"))
 
+if sys.version_info < (3, 9):
```

Comment:
Plz move this to a constant in `helpers.py`. Could call it `PY_OLDER_THAN_39`.

Reply:
Alternatively, have an inverted name
```suggestion
if not PY_39_PLUS:
```

Reply:
I'd prefer if we didn't. That often results in more work for me when dropping versions, and often doesn't play well with mypy (I suspect I may have removed a previous PY_39 constant or similar previously).